.github/workflows/cmake.yml

```diff
@@ -0,0 +1,90 @@
+# inspired by "build GDAL library and to run test with CMake"
+# from https://github.com/OSGeo/gdal/blob/master/.github/workflows/cmake_builds.yml
```

Comment:
Please replace this comment by a description what is actually happening here and why. This is perhaps for the commit message, but I find this info here distracting. What should I do with that >700 lines file linked from here? The current comment is not actionable.